contracts/app/andromeda-app-contract/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "andromeda-app-contract"
-version = "1.0.1"
+version = "1.1.1"
```

Should just be `1.1.0`

```suggestion
version = "1.1.0"
```